pkg/apiserver/rest/usecase/config.go

```diff
@@ -321,6 +345,11 @@ func SyncConfigs(ctx context.Context, k8sClient client.Client, project string, t
 	}
 
 	mergedTarget := mergeTargets(currentTargets, targets)
```

Comment:
I can get through here temporarily, but incrementally increasing delivery targets is still a risky behavior.